### DISCUSSION TITLE
FPS keep changing

### DISCUSSION THREAD

**Discussion by krisadamqdc** [pedroSG94/RootEncoder#950] [Q&A]
I need some help here. We use rtmpcamera on android but we are not sure why FPS keep changing while live stream. We monitor on live stream platform and showing FPS using listener in the library to monitor the FPS but same result that FPS is not stable. 

Anything we can do or anyone have this problem? Anything we need to do to have it stable?

https://user-images.githubusercontent.com/80890688/135800809-db3fc2a7-90b6-44b6-8f9f-9f1826337f33.mov

**Comment by pedroSG94** [comment]
The fps will be the closest value to your fps value set in preparevideo depend of your internet bandwidth (if you have a poor connection the library could discard frames) and your device performance (if you can't handle more fps for hardware reasons). If you have a constant fps using 30fps but the fps is not working fine with lower values, this could be a library bug. In this case let me know:
Mode used (camera1, display, etc), view used (openglview, surfaceview) and preparevideo configuration.

To test it try set fps to low value (20 or 15 for example) and check if this keep constant using OpenGlView.
